core/src/main/java/io/confluent/rest/metrics/MetricsResourceMethodApplicationListener.java

```diff
@@ -173,6 +175,12 @@ public ConstructionContext(
     private Sensor responseSizeSensor;
     private Sensor requestLatencySensor;
     private Sensor errorSensor;
+    private Sensor errorSensorUnknown;
+    private Sensor errorSensor100s;
```

Comment:
you could refactor these into a simple array, which makes the code for deciding which sensor it should go an integer division and fewer conditionals. we should take care about performance in this code since it has overhead per-request (at least, in this case, for error cases).